File: javascript/1048.js
var input = require('fs').readFileSync('/dev/stdin', 'utf8');
const s = parseFloat(input);
var news;
var plus;
var perc;
if(s <= 400){
     news=s+s*0.15
     plus=s*0.15
     perc=15
}else if(s <= 800){
     news=s+s*0.12
     plus=s*0.12
     perc=12
}else if(s <= 1200){
     news=s+s*0.1
     plus=s*0.1
     perc=10
}else if(s <= 2000){
     news=s+s*0.07
     plus=s*0.07
     perc=7
}else if(s > 2000){
     news=s+s*0.04
     plus=s*0.04
     perc=4
}
console.log(`Novo salario: ${news.toFixed(2)}`);
console.log(`Reajuste ganho: ${plus.toFixed(2)}`);
console.log(`Em percentual: ${perc} %`);

File: python/1048.py
s=float(input())
sf=s
pc=0
if s >= 0 and s <= 400:
    sf+= s*0.15
    pc=15
elif s >= 400.01 and s <= 800:
    sf+= s*0.12
    pc=12
elif s >= 800.01and s <= 1200:
    sf+= s*0.1
    pc=10
elif s >= 1200.01 and s <= 2000:
    sf+= s*0.07
    pc=7
elif s > 2000:
    sf+= s*0.04
    pc=4
    
rg=sf-s

print("Novo salario: "+str('{:.2f}'.format(sf)))
print("Reajuste ganho: "+str('{:.2f}'.format(rg)))
print("Em percentual: "+str(pc)+" %")